auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -553,6 +554,10 @@ public boolean isLongTimeBackupEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_DATALAKE_BACKUP_LONG_TIMEOUT);
     }
 
+    public boolean isPostgresUpgradeAttachedDatahubsCheckEnabled(String accountId) {
+        return isEntitlementRegistered(accountId, CDP_POSTGRES_UPGRADE_ATTACHED_DATAHUBS_CHECK);
```

Comment:
This could become:
CDP_POSTGRES_UPGRADE_SKIP_ATTACHED_DATAHUBS_CHECK

In this way the logic also changes: running datahub check is performed unless the user has this entitlement is granted, to lift the requirement for checking DHs.
This would also change the PRs logic and some tetst.